pkg/shell/nav.scss

```diff
@@ -171,6 +171,7 @@ $desktop: $phone + 1px;
 }
 
 [dir="rtl"] {
+
```

Comment:
Likewise, there's several spurious newlines which don't belong here. Please revert. The only relevant change for the caret is the introduction of the `.clicked` class.